Week-1/Advanced/index.html

```diff
@@ -0,0 +1,98 @@
+    <p>UA College of Pharmacy | Aug. 6, 2014</p>
+    <p>
+      Arguably, snake season is year-round in Arizona, a state known for its
+      rattlers. But baby rattlesnakes are born in July and August, making these
+      two months especially dangerous for hikers, gardeners, children and others
+      at high risk of exposure to rattlesnake bites.
+    </p>
+    <img
+      src="./SnakeBite.jpg"
+      alt="Snake bite on side of child's foot.
+    "
+    />
+    <p>
+      So far this year, 74 rattlesnake bites to humans have been reported to the
+      Arizona Poison and Drug Information Center. Based at the University of
+      Arizona College of Pharmacy, the center serves the entire state of Arizona
+      with the exception of Maricopa County, providing free and confidential
+      poison and medication information to callers around the clock.
+    </p>
+    <p>
+      Specialists answering the phones at the center regularly receive calls
+      from Arizonans of all ages who don't realize they were bitten by a
+      rattler. The poison center urges anyone who feels a mysterious sting,
+      pinch or bite while outdoors to immediately call the center at
+      800-222-1222.
+    </p>
+    <p>
+      "We will ask a few questions that will help you either identify possible
+      snakebite or eliminate it," said Keith Boesen, director of the Arizona
+      Poison and Drug Information Center. "With snakebite, the sooner the
+      medical treatment, the better the outcome, so calling us right away can
+      make a very big difference for the victims and the medical teams treating
+      them."
+    </p>
+    <p>
+      The center advises anyone who might come cross paths with rattlesnakes to
+      be aware of these five things:
+    </p>
+    <!-- unordered list of paragraphs -->
+    <ol>
+      <li>
+        <p>
+          Baby rattlesnakes range in length from 6 to 12 inches and are easily
+          camouflaged by brush and grass
+        </p>
+      </li>
+      <li>
+        <p>
+          Baby rattlesnakes are rattleless until they first shed their skins, so
+          there will be no infamous "chica-chica" sound before they strike.
+        </p>
+      </li>
+      <li>
+        <p>
+          Despite their impish size, baby snakes have enough venom to be very
+          dangerous if they bite a human.
+        </p>
+      </li>
+      <li>
+        <p>
+          Adult rattlesnakes do not always rattle an audible warning before or
+          while they are biting.
+        </p>
+      </li>
+      <li>
+        <p>
+          It's a good idea to call the poison center if you notice an
+          unidentified small cut or wound, even if you feel no pain. With the
+          lack of telltale rattle warning, people can be bitten without knowing
+          what has happened until they notice their symptoms and attribute them
+          to a snakebite.
+        </p>
+      </li>
+    </ol>
```

Comment:
we can footer tag here, we should keep the structure ideally as head, body,footer . each can comprise child elements.
https://www.w3schools.com/tags/tag_footer.asp